src/test/java/hudson/plugins/active_directory/docker/TheFlintstonesTest.java

```diff
@@ -101,6 +115,56 @@ public void simpleLoginFails() throws Exception {
         }
     }
 
+    @Issue("JENKINS-45576")
+    @Test
+    public void loadGroupFromGroupname() throws Exception {
+        String groupname = "The Rubbles";
+        GroupDetails group = j.jenkins.getSecurityRealm().loadGroupByGroupname(groupname);
+        assertThat(group.getName(), is("The Rubbles"));
+    }
+
+    @Issue("JENKINS-45576")
+    @Test
+    public void loadGroupFromAlias() throws Exception {
+
+        // required to monitor the log messages, removing this line the test will fail
+        List<String> logMessages = captureLogMessages(20);
+
+        String aliasname = "Rubbles";
+        boolean isAlias = false;
+        try {
+            j.jenkins.getSecurityRealm().loadGroupByGroupname(aliasname);
+        } catch (Exception e) {
+        } finally {
+            Collection<String> filter = Collections2.filter(logMessages, Predicates.containsPattern("JENKINS-45576"));
+            isAlias = !filter.isEmpty();
+        }
+
+        assertTrue(isAlias);
+    }
+
+    private List<String> captureLogMessages(int size) {
+
```

Comment:
Would you mind to delete this empty line?

Reply:
done